package.json

```diff
@@ -48,13 +48,13 @@
 		"test": "npx htest ./test",
 		"dtslint": "dtslint --expectOnly types",
 		"eslint": "eslint . --ext .js --ext .ts --ext .cjs",
-		"lint": "run-s build:space-accessors \"eslint -- --fix\" dtslint",
-		"lint:ci": "run-s build:space-accessors \"eslint -- --max-warnings 0\" dtslint",
+		"lint": "run-s build:space-accessors \"eslint -- --fix\" build:ts dtslint",
+		"lint:ci": "run-s build:space-accessors \"eslint -- --max-warnings 0\" build:ts dtslint",
```

Comment:
I think `build:ts` should run in an earlier step to make sure that all the files are in place before `dtslint` runs. Maybe add a build step to the CI file, after `npm install` and before `npm run lint:ci`?

```yaml
            - name: Generate TypeScript definitions
              run: npm run build:ts
```

Reply:
Nevermind—I confused `run-s` with `run-p`